ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -2,3 +2,46 @@
 # defaults file for installer
 cache_enabled: false
 provision_cache_store: "{{ ansible_env.HOME }}/rhcos_image_cache/"
+
+# Information used to create certificates when creating the registry.
+cert_country: MyCountry
+cert_state: MyState
+cert_locality: MyCity
+cert_organization: MyCompany
+cert_origanizational_unit: MyBusinessUnit
+
+# A file that contained extra auths to be included in the pull secret.
+disconnected_registry_auths_file:
+
+# Name of a file containing the additionalTrustBundle and mirrored
+#   repository settings. If this is set, then a local registry will not be created.
+#   Setting this causes the playbook to use an existing disconnected registry.
+disconnected_registry_mirrors_file:
+
+# The containers registry port
+registry_port_container: 5000
+
+# The port exposed on the server for the registry
+registry_port: "{{ registry_port_container }}"
```

Comment:
Same comment as above: Since right now we hardcode opening this specific port (5000), I don't think we should offer this value as something that is changeable. Instead I think this variable should exist in installer/vars/main.yml

Reply:
same response as above

Reply:
Let me know if you agree with comment above here since its the same for this one.